### DISCUSSION TITLE
why when I use this plugin with toggleterm and lf it seems weird?

### DISCUSSION THREAD

**Discussion by JuanZoran** [willothy/flatten.nvim#50] [Q&A]
It seems doesn't load my vim.opt conf, bacause I lost my line number ....

![image](https://github.com/willothy/flatten.nvim/assets/107862700/74b5f31f-4268-4bb4-ae60-51edfdd79ec5)
this is my configuration for flattern.nvim
> btw, I think I should use `vim.cmd.q` instead of require'toggleterm'.toggle(0), because when I use the new toggleterm instance, it doesn't wok, maybe this should be added into the README file...
![image](https://github.com/willothy/flatten.nvim/assets/107862700/7e668a30-b0ae-43bc-9967-67ba04b789e2)

**Comment by willothy** [comment]
I'll try toggleterm again, I'm using NVterm at the moment but haven't had issues with toggleterm in the past. I'll update here after testing.

**Comment by willothy** [comment]
I use toggleterm now and have for months, haven't had this problem. If you're still experiencing it, please open a bug report. Thanks!
